src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Threading;
+
+namespace System.Device.Pwm.Channels
+{
+    /// <summary>
+    /// Represents a PWM channel running on Unix.
+    /// </summary>
+    internal class UnixPwmChannel : PwmChannel
```

Comment:
@krwq do you think we should make this and WindowsPwmChannel sealed?

Reply:
i don't care, it's internal